src/components/_shared/styled-headings.js

```diff
@@ -1,11 +1,8 @@
 import styled from '@emotion/styled';
 
 export const StyledH1 = styled.h1`
-  width: 100%;
```

Comment:
why are the changes needed? I actually think you should rather revert them, because the relative is required for that pseudo element on our headings